src/accessories/platformAccessory.ts

```diff
@@ -13,7 +14,60 @@ export abstract class MiHomePlatformAccessory {
     protected readonly platform: MiHomeGatewayPlatform,
     protected readonly accessory: PlatformAccessory,
   ) {
-    // TODO
+
+    // set accessory information
+    this.accessory.getService(this.platform.Service.AccessoryInformation)!
+      .setCharacteristic(this.platform.Characteristic.Manufacturer, MANUFACTURER)
+      .setCharacteristic(this.platform.Characteristic.Model, this.accessory.context.device_type)
+      .setCharacteristic(this.platform.Characteristic.Name, this.accessory.context.label)
+      .setCharacteristic(this.platform.Characteristic.AppMatchingIdentifier, APP_MATCHING_IDENTIFIER);
```

Comment:
What does this do?

Reply:
This sets the accessory information, including the `AppMatchingIdentifier` which matches the accessory to an app in the App Store.

Reply:
Ah so this will link the accessory to the MiHome app? That's neat. Where did you find out about this? Would be keen to read up on little bits like this...

Reply:
I have an Apple Developer account and got the information from their, the Homebridge developer website is also a useful resource: https://developers.homebridge.io/#/characteristic/AppMatchingIdentifier

Reply:
Ah same, didn't even think to look there - will had a read :)

Reply:
It can be a little awkward to navigate, but it's got the most information. :)